apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.tsx

```diff
@@ -12,14 +12,19 @@ const GetInvolved = () => {
 		<p>Otherwise, if you have some experience under your belt,</p>
 	);
 	const applyLink = (
-		<Button href="/apply" type="button" className={styles.applyButton}>
-			Apply as a Hacker
+		<Button href="/apply" type="button" className={styles.applyButton} disabled>
```

Comment:
href still here (already mentioned but still noting)

also, should the button still have a hover state even when apps are closed?

Reply:
The `/apply` route redirects to home anyways, so I think it's fine.

The hover state is still there for style. It doesn't look like the hover state for when the button is active. It gives greyed out feedback, which I think looks better than a simple disabled button.

Same thing for the svgs, it'll just change to the greyed out disabled button.